components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/token/handlers/grant/AuthorizationCodeGrantHandler.java

```diff
@@ -306,6 +316,25 @@ private void revokeExistingAccessTokens(String tokenId, AuthzCodeDO authzCodeDO)
                         "active. So revoking the access tokens issued for the authorization code.");
             }
         }
+        invokePostAccessTokenRevocationListeners(accessTokenDO);
+    }
+
+    /**
+     * Invokes Post Access Token Revocation Listeners
+     * @param accessTokenDO nullable AccessTokenDO
+     */
+    private void invokePostAccessTokenRevocationListeners(AccessTokenDO accessTokenDO) {
+        if (accessTokenDO != null) {
```

Comment:
We can take the inverse of this else block and stop code block getting shifting to the right. WDYT?

Reply:
Fixed with 33f8ddecc5895779fb2893cccb3f528d456fed43